### ISSUE TITLE
DOCTOR function is in the code, but it never gets called

### ISSUE THREAD

**Issue by darth-crunchus** [drugwars-sh/drugwars.sh#2]
I was wondering whether I was going to be able to get healed or not, so I searched the script for a call to the DOCTOR function, and the only two instances found are the name of the function itself and the word "DOCTOR" inside the script.  Where should the function be called?  Somewhere in the main menu, after fights?

**Comment by drugwars-sh** [comment]
Yeah I remember adding some of it, but I just never figured out exactly how it worked, e.g. the price in the original game.
